content/applications/inventory_and_mrp/barcode/setup/hardware.rst

```diff
@@ -16,24 +16,27 @@ Follow this guide to choose and set up a barcode scanner compatible with Odoo's
 Scanner types
```

Comment:
I know that you didn't change the first 15 lines of this doc, but I'd suggest removing the text in line 14, as that alt text is already in the image formatting in line 12.

Reply:
I would also rename this doc to remove the verb as the first word...

something like: "Barcode scanner setup" could be a could option